README.md

```diff
@@ -1,19 +1,36 @@
 # scalar-admin
 
-scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications.
+scalar-admin is an admin interface and client tool for Scalar services such as [Scalar DL](https://github.com/scalar-labs/scalardl) and [Scalar DB](https://github.com/scalar-labs/scalardb) applications. 
+It will help you to `PAUSE` and `UNPAUSE` the ledger to create point-in-time recovery (PITR). 
 
 ## Use the client-side tool
 
-To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows.
+To access scalar-admin integrated applications, you can use the fat jar `scalar-admin-<version>-all.jar` found in [releases](https://github.com/scalar-labs/scalar-admin/releases) as follows
+
+```console
+$ java -jar scalar-admin-<version>-all.jar -c <Command> -s <SRV_Service_URL>
+```
+
+If you want to use the scalar-admin docker container, you can do it as follows.
```

Comment:
We have never explained the docker container, so `the scalar-admin docker container` sounds weird.
We should explain we have a fat jar and a container first, then explain the usage. Now they are mixed up.

Why don't you have the following structure? (separating the installation from the usage)
Please think about the contents by yourself.
```
## Installation
### Jar (please update appropriately)
### Container (please update appropriately)

## Usage
### Jar (please update appropriately)
### Container (please update appropriately)
```

Reply:
Thank you for your suggestion.
Document updated with the above structure.

I think the Installation step is not required here, because the flat jar is available in the release tag and the docker image is available in the docker registry. (It is available in the current doc because I am not sure about this)